pkgs/applications/blockchains/btcpayserver/default.nix

```diff
@@ -37,21 +35,15 @@ stdenv.mkDerivation rec {
     nuget init ${linkFarmFromDrvs "deps" deps} $TMP/nuget
 
     dotnet restore --source $TMP/nuget BTCPayServer/BTCPayServer.csproj
-    dotnet build -c Release BTCPayServer/BTCPayServer.csproj
-  '';
-
-  runScript =  ''
-    #!${bash}/bin/bash
-    DOTNET_CLI_TELEMETRY_OPTOUT=1 exec ${dotnetSdk}/bin/dotnet run --no-launch-profile --no-build \
-      -c Release -p @@SHARE@@/BTCPayServer/BTCPayServer.csproj -- "$@"
+    dotnet publish --no-restore --output $out/share/$pname -c Release BTCPayServer/BTCPayServer.csproj
```

Comment:
```suggestion
    DOTNET_CLI_TELEMETRY_OPTOUT=1 dotnet publish --no-restore --output $out/share/$pname -c Release BTCPayServer/BTCPayServer.csproj
```

Reply:
`DOTNET_CLI_TELEMETRY_OPTOUT` is already set [here](https://github.com/erikarvstedt/nixpkgs/blob/update-btcpayserver/pkgs/applications/blockchains/btcpayserver/default.nix#L32).